src/Components/Header/ThemeSwitch/ThemeSwitch.jsx

```diff
@@ -1,16 +1,32 @@
-import { useContext } from 'react';
-import { ThemeContext } from '../../../context/themeContext';
+import { useSelector, useDispatch } from 'react-redux';
+import { toggleTheme } from '../../../redux/theme';
 
 import styles from './ThemeSwitch.module.scss';
 
 function ThemeSwitch() {
-  const { theme, toggleTheme } = useContext(ThemeContext);
+  const theme = useSelector((state) => state.theme);
+  const dispatch = useDispatch();
 
   return (
     <div className={styles['theme-switch']}>
       <div className={styles.wrapper}>
         <label className={styles.switch}>
-          <input type="checkbox" id="checkbox-toggle" onClick={toggleTheme} />
+          {theme === 'light' ? (
+            <input
+              type="checkbox"
+              defaultChecked={true}
+              id="checkbox-toggle"
+              onClick={() => dispatch(toggleTheme())}
+            />
+          ) : (
+            <input
+              type="checkbox"
+              defaultChecked={false}
+              id="checkbox-toggle"
+              onClick={() => dispatch(toggleTheme())}
+            />
```

Comment:
why not like this? or just make this input controlled
```
<input
  type="checkbox"
  checked={theme === "light"}
  id="checkbox-toggle"
  onClick={() => dispatch(toggleTheme())}
/>
```

Reply:
ok, fixed